src/views/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <meta property="og:image" content="https://github.com/MizouziE/energy-prices-api/blob/6f1a91d79f2da16da4ccbc3b86ad6b0d7134606a/public/img/energy-prices-api-socials.png">
```

Comment:
Great stuff and also great thinking regarding the extra meta tags with dimensions! I will wait for their addition to this branch before merging. 

Thanks a lot! 😁